roles/confluent.test/molecule/rbac-mtls-rhel/verify.yml

```diff
@@ -375,3 +376,46 @@
           - schema_registry_name == "{{cluster_name_query_json[3]['clusterName']}}"
         fail_msg: "Schema Registry Cluster ID set to {{cluster_name_query_json[3]['clusterName']}} not {{schema_registry_name}}"
         quiet: true
+
+- name: Validate SSL mapping rule
+  hosts: localhost
+  gather_facts: false
+  tasks:
+    - import_role:
+            name: confluent.variables
```

Comment:
this seems like a net new feature, should it go into 7.1.x?

Reply:
The original issue was created for 6.2.1 so the fix is going here in next patch release.